packages/uma-auth-client/src/components/ConnectionCard.tsx

```diff
@@ -0,0 +1,107 @@
+import styled from "@emotion/styled";
+import { Button, Icon } from "@lightsparkdev/ui/components";
+import { Label } from "@lightsparkdev/ui/components/typography/Label";
+import { Connection, Currency, LimitFrequency } from "src/types/connection";
+import { formatAmountString } from "src/utils/currency";
+
+interface Props {
+  connection: Connection;
+  uma?: string | undefined;
+  balance?:
+    | {
+        amountInLowestDenom: number;
+        currency: Currency;
+      }
+    | undefined;
+}
+
+export const ConnectionCard = ({ connection, uma, balance }: Props) => {
+  const handleCopyUma = () => {
+    navigator.clipboard.writeText(uma || "");
```

Comment:
Should we do the little "copied" notification thing we do on cn2 here?

Reply:
Yeah i think so, or just a change in icon/animation will probably work so it's not as heavy. Will ask design